### ISSUE TITLE
[SUGGESTION] - Active Warrants Section

### ISSUE THREAD

**Issue by GodyFromDiscord** [teddythedev/legacy-mdt-issue-tracker#13]
**Summary:**
For the active warrants section, I believe we should have a little button like the Submit one when creating a new incident to add on a warrant, now the following versions could be done until more advanced development is done;

Basic - Have an Add/Create section where it prompts you to insert their CID, and link the warrant document itself, then on the front of the page it will show the person's name, date of warrant submitted and who added it, and then clicking on the button takes you to the warrant document like below;

![image](https://i.imgur.com/SOjyjb8.png)

Maybe have two extra icons inside or next to the box, one of a person icon, which takes you to their profile and a paper icon which opens the warrant document. To remove a warrant any officer can press a Bin icon to get rid of it, these are tracked.

Advanced - The officer creates the warrant document as normal, this is then submitted but not shown to everyone just yet, Chiefs will see the warrant but in a grey box like below; 

![image](https://user-images.githubusercontent.com/60844168/189565074-153f9015-d792-41b3-91af-9483dd9af6bb.png)

This could be in a pending warrants box, this takes the Chief to the document where they sign it off and then can tick or deny on the MDT which published it to every other officer to see on the MDT

---

Overall as well it would be nice to have a little icon or signifier when going onto someone's profile if they have a warrant or not and prompts them to the warrants page

**Reason:**
Makes a cool use of the Active warrants section :)

**Comment by teddythedev** [comment]
Yeah this is my next project. It is going to pretty much be just like you described above!
